ebu_tt_live/xsd/ebutt_metadata.xsd

```diff
@@ -472,26 +473,11 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 				</xs:attribute>
 			</xs:extension>
 		</xs:simpleContent>
-		
-	</xs:complexType>
-	
-    <xs:complexType name="trace_type">
-		<xs:annotation>
```

Comment:
Would be good to add an equivalent annotation section to the `appliedProcessing` element too.